.php_cs

```diff
@@ -0,0 +1,15 @@
+<?php
+
+return Symfony\CS\Config\Config::create()
+    ->fixers([
+        '-phpdoc_short_description',
```

Comment:
IMHO, those fixers should not be removed.

As a Symfony bundle, we should respect all Symfony standards.